EC-84: Restructure eZ Commerce doc, part 2 (#2)
ezsystems/commerce-documentation: docs/guide/basket/basket_features/additional_data_in_the_basket_line.md

![](../../img/basket_additional_data_1.png)

> To set a different value for the parameter there are two possibilities.

Reviewer: German syntax probably. I would suggest rewriting the sentence:
```suggestion
You can set a different value for the parameter by using the following two methods: 
```

## Additional data

> To add some additional information to the basket line **only the template** will be modified. No other changes are necessary.

Reviewer: Does this mean the following?
To add more information to the basket line, you modify **the template only**.
I found it difficult to grasp the meaning of the sentence

> To add some additional information to the basket line **only the template** will be modified. No other changes are necessary.

Reviewer 2: ```suggestion
You can add additional information to the basket line ** by modifying the template**. No other changes are necessary.
```
